azext_edge/edge/providers/check/dataflow.py

```diff
@@ -54,6 +55,146 @@
+        provisioning_status_error = provisioning_status.get("error")
+        provisioning_status_failure_cause = provisioning_status.get("failureCause")
+        provisioning_status_enum = (
+            CheckTaskStatus.error
+            if provisioning_status_error or provisioning_status_failure_cause
+            else ResourceState.map_to_status(provisioning_status_status)
+        )
+
+        provisioning_status_display = f"Provisioning Status: {{{colorize_string(color=provisioning_status_enum.color, value=provisioning_status_status)}}}"
+
+        # if not summary, display output / message
+        if detail_level > ResourceOutputDetailLevel.summary.value:
+            provisioning_status_output = provisioning_status.get("output", {})
+            # output object has no schema, but expecting "message" property to have details
+            provisioning_status_output_message = provisioning_status_output.get("message")
+            # if no "message" property, just use the entire object as output
+            provisioning_status_output_display = provisioning_status_output_message or provisioning_status_output
+
+            provisioning_status_display += f" - {colorize_string(provisioning_status_output_display)}"
+
+        # status display
+        check_manager.add_display(
+            target_name=target_name,
+            namespace=namespace,
+            display=Padding(provisioning_status_display, (0, 0, 0, padding)),
+        )
+
+        # display error, failure cause
+        if provisioning_status_error:
+            message = provisioning_status_error.get("message")
+            code = provisioning_status_error.get("code")
+            error_text = f"{code}: {message}"
+            error_display = f"Error: {{{colorize_string(color=CheckTaskStatus.error.color, value=error_text)}}}"
+            check_manager.add_display(
+                target_name=target_name,
+                namespace=namespace,
+                display=Padding(error_display, (0, 0, 0, padding))
+            )
+        # show failure cause
+        if provisioning_status_failure_cause:
+            check_manager.add_display(
+                target_name=target_name,
+                namespace=namespace,
+                display=basic_property_display(
+                    label="Failure Cause", value=provisioning_status_failure_cause, padding=inner_padding
+                ),
+            )
+
+        # logErrors on detail level 1
+        provisioning_status_log_errors = provisioning_status.get("logErrors")
+        if detail_level > ResourceOutputDetailLevel.summary.value and provisioning_status_log_errors:
+            check_manager.add_display(
+                target_name=target_name,
+                namespace=namespace,
+                display=Padding("Log Errors:", (0, 0, 0, inner_padding)),
+            )
+
+        # operation ID on detail level 2
+        provisioning_status_operation_id = provisioning_status.get("operationId")
```

Comment:
we were talking about this offline, if we want to have this operation id, wondering if making the value into its own line would be better visually?

ex:
Opertation ID:
    abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd...

Reply:
It's very awkward to display - chatting offline to determine whether this is even worth displaying or just removing from the output.